openapi/components/schemas/GatewayAccountConfig/Tap.yaml

```
@@ -15,3 +15,10 @@ allOf:
             format: password
         required:
           - apiKey
+      settings:
+        type: object
+        description: Tap settings object
+        properties:
+            use3DSecure:
```

```suggestion
            enable3DSecure:
```

Just wonder which language is better?

---

@slavcodev, in recomm, there's `Enable 3D Secure` option which signifies that we handle the 3Ds process on our end. In Tap's case, they have their own 3D Secure process which can be toggled via just a request flag. I'm actually torn between `use3DSecure` vs `useGateway3DSecure`. The latter says specifically to use the gateway's 3D Secure.